src/com/MC/ticTacToe/controller/Game.java

```diff
@@ -0,0 +1,82 @@
+package com.MC.ticTacToe.controller;
+
+import com.MC.ticTacToe.helper.TicTacToeHelper;
+import java.util.Scanner;
+
+public class Game {
+
+    int noOfPlayers;
+    int noOfCells;
+    String[][] players;
```

Comment:
Rename to something specific? This seems very generic and doesn't provide info on what this 2D array means.